src/behaviours/Tweened/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { PureComponent } from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { omit, pick } from 'lodash';
+import uuid from 'uuid';
+import getAbsoluteBoundingRect from '../../utils/getAbsoluteBoundingRect';
+import tween from './tween';
+
+window.Tweens = window.Tweens || [];
```

Comment:
Since this is all 'out of' react anyway, limited use of window seems fine to me. But if it's bothering you, would a separate redux store work? (Would help debugging, if nothing else.)